### ISSUE TITLE
Meteor-embedded underscore library contains vulnerabilities

### ISSUE THREAD

**Issue by carlosalvidrez** [meteor/meteor#12673]
Hi guys, is there a way for me to upgrade the meter/underscore library to a more modern version? The one that seems to be bundled with meteor 2.12 is being reported by Chrome's Lighthouse as vulnerable (attached screenshot). 

Meteor Version 2.12.
Chrome Lighthouse inspection reports a vulnerability in the underscore library.
I have it added as "underscore" in the packages file.

![image](https://github.com/meteor/meteor/assets/1237440/e2da957a-730c-4cbf-b845-1484d366c650)

**Comment by jamauro** [comment]
There was an effort to remove underscore. See: https://github.com/meteor/meteor/pulls?q=is%3Apr+is%3Aopen+remove+underscore

Sounds like those PRs should be merged

**Comment by carlosalvidrez** [comment]
Thanks @jamauro ... I am also using the library on some of my custom code routines... do you have any suggestion as to a good replacement library? Unless newer versions of underscore, that are not vulnerable, are available, I might have to switch. Thanks

**Comment by jamauro** [comment]
Take a look at lodash or you might be able to just use native JS https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore

**Comment by StorytellerCZ** [comment]
@Grubba27 I think we need to merge as many of the remove underscore PRs as possible for the next release. In the meantime I will update our version of underscore to the latest release before underscore was split from the single file. Already got v1.6 done.

Just a note to everyone, Meteor has adjusted underscore in small parts so that needs to be taken into account when doing these upgrades and sadly it isn't as simple as just upping a version number.

**Comment by StorytellerCZ** [comment]
@carlosalvidrez do you know what minimum underscore version we need to clear the warning?

**Comment by carlosalvidrez** [comment]
Hi @StorytellerCZ! It's this:

`underscore@1.0.13`

**Comment by StorytellerCZ** [comment]
@carlosalvidrez that is strange 1.5.2 > 1.0.13

If it is 1.13.0, then that would make sense, but that one will be quiet difficult as I think that is after the split into multiple packages.

**Comment by carlosalvidrez** [comment]
Yes, it's strange, puzzled as to why Chrome detects 1.5.2... weird!
Perhaps it's a duplicate dependency of an npm package?

The version I quoted above is from the meteor "packages" file.

**Comment by StorytellerCZ** [comment]
The Meteor packages internally uses underscore 1.5.2.

**Comment by Grubba27** [comment]
Hey @StorytellerCZ, I agree! could you tag those green underscore PRs so we can work on them for the 2.14 release?

**Comment by StorytellerCZ** [comment]
I'll add them to the milestone.

**Comment by StorytellerCZ** [comment]
Meteor 2.15, updates underscore package to 1.6 (after that I'm having some issues which I need to debug). Will look if there is some other place where we get underscore.

**Comment by italojs** [comment]
still relevant https://security.snyk.io/package/npm/underscore
